run0/src/main.rs

```diff
@@ -149,6 +149,20 @@ impl Runtime {
 
         Ok(())
     }
+
+    fn get_env_variables(&self) -> Result<Vec<(&str, &str)>> {
```

Comment:
Rust idiom for a getter for field `foo` is `foo()`. So I'd call that method `env_variables` instead.